meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -1,7 +1,7 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
 BRANCH_lmp = "master"
-SRCREV_lmp = "5e2f6da2c1ad02ea87fa32564d9b37d9771a4ec3"
+SRCREV_lmp = "faaee8dc694be8a1edae6b9f06e9b4c5743fa744"
```

Comment:
Rev is invalid, seems to be https://github.com/foundriesio/aktualizr-lite/commit/ac6df2fd1ce1bf4f4fe80364dc9e3931791254b6 instead. Maybe we should bump to https://github.com/foundriesio/aktualizr-lite/commit/201341a3c1d8c1d09cea07bb1f74916fabea0db5 now?

Reply:
rebased and corrected the aklite ref.